Dockerfile

```diff
@@ -69,7 +69,7 @@ COPY --from=mozilla/ingestion-sink:latest /app/ingestion-sink/target /app/target
 
 # Install python dependencies
 COPY requirements.txt requirements-dev.txt ./
-RUN pip3 install --upgrade pip setuptools && \
+RUN pip3 install --upgrade pip setuptools==70.3.0 && \
```

Comment:
Does the newer setuptools cause problems running things locally?  If so, then I'd think you'd also want to pin setuptools in `requirements.txt`.

Reply:
I'm not sure we should add setuptools to requirements: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/36abca46ba9a311628ef8cde510e9f74becba16a/requirements-dev.txt#L28C3-L29

Reply:
Under normal circumstances I agree, but if the setuptools version incompatibility was going to consistently cause problems for devs that would take precedence IMO.

Luckily this is a moot point at the moment because it turns out the latest production versions of Python only come with setuptools 65 installed by default (and Python 3.8, which is the version used in the Dockerfile, comes with setuptools 56 by default).